app/models/time_data/required_time_data.rb

```diff
@@ -0,0 +1,19 @@
+module TimeData
+
+  class RequiredTimeData
+
+    def order_completable?
+      false
+    end
+
+    def blank?
+      true
+    end
+
+    def problem?
```

Comment:
This is a small thing, but it would be nice if the methods on `RequiredTimeData` were in the same order as in `NullTimeData`, just so it's easier to see the differences.